ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -933,7 +934,8 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
     }
     out_mem = gst_buffer_get_all_memory (outbuf);
   }
-  g_assert (gst_memory_map (out_mem, &out_info, GST_MAP_WRITE));
+  status = gst_memory_map (out_mem, &out_info, GST_MAP_WRITE);
+  g_assert (status);
```

Comment:
Instead of g_assert(), how about 
```c
if (status == FALSE)
 return GST_FLOW_ERROR;
```

Reply:
That'd be yet another topic, requiring yet another commit.